.not-formatted

```diff
@@ -1,8 +1,11 @@
-./ext/lmdb-safe/lmdb-safe.cc
```

Comment:
`make format-code` tries to format these vendor files if you remove these `ext` lines...

Reply:
I see. I keep forgetting to look at `make format-code` because I don't use it at all.

However, do vendored files really fall under our responsibility? Doesn't formatting them just make it more work everytime we want to update the dependencies?

I guess my question is about whether we should disable the formatting of vendored files in `make format-code`.